src/handler.rs

```diff
@@ -148,7 +155,32 @@ pub enum Command {
 
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct SubstrateCommand {}
+pub enum SubstrateCommand {
+    MixerRelayTx(MixerRelayTransaction),
+}
+
+#[derive(Debug, Clone, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MixerRelayTransaction {
+    /// one of the supported chains of this relayer
+    pub chain: String,
+    /// The tree id of the mixer's underlying tree
+    pub id: u32,
+    /// The zero-knowledge proof bytes
+    pub proof: Vec<u8>,
+    /// The target merkle root for the proof
+    pub root: Element,
```

Comment:
The type `Element` will not work here, as it does not implement serde traits.

Prefer the use of `H256` it will give you the exact 32 bytes.
```suggestion
    pub root: H256,
```